### DISCUSSION TITLE
MQA, DSD, etc. playback

### DISCUSSION THREAD

**Discussion by AECX** [jeffvli/feishin#401] [General]
Hello everyone,

I just got a new [DAC](https://www.audiosciencereview.com/forum/index.php?threads/smsl-d12-dac-headphone-amp-review.46521/) and wondered why my output behaves differently when playing a file through Feishin.

Since Feishin is using mpv I manuall played the file via command line "mpv --audio-exclusive=yes file.mqa.flac" which worked like a charm, the DAC is correctly detecting the MQA file and playing it properly.

So then I uploaded that same file to my Jellyfin server and played it through Feishin, with custom arguments, the preset "exclusive mode" and "sample rate", with a restart each change. But it didn't work.

Sure the exclusive mode and sample rate were applied but when checking the DAC it doesn't detect the MQA file as if I was playing it locally with the exact same MPV assembly.

First I checked whether my Jellyfin instance is transcoding the file, it is not. So then I checked the command line parameters for any differences but the only ones are things like the play queue (duh) and "--no-config" etc. Didn't seem like a problem to me and afaik there's no option to disable the default parameters in Feishin is there?

I would appreciate any help here, thank you very much.

Kind Regards
AECX

**Comment by jeffvli** [comment]
The first thing I would try is to use mpv to play the direct stream from Jellyfin via the CLI like you did with the file.

If you go to the Jellyfin Web UI, you can select the ellipsis dropdown next to a song and select `Copy Stream URL` which will copy the direct stream link to your clipboard.

Then try running the same mpv with the URL instead of the file to see if you get your expected behavior.

If it works as expected, then we can look into setting some different mpv defaults in Feishin, but if not it may be an issue on how Jellyfin serves the file.

**Reply by AECX** [reply]
Thanks! I didn't think of that but it worked well. My DAC MQA indicator is lighting up just like with local playback and exclusive mode.

Let me know if you need any information about my setup or if I can help in any other way.

Cheers

**Comment by AECX** [comment]
bump

**Reply by jeffvli** [reply]
Unfortunately I'm not sure exactly what the issue could be. Have you tried using another audio server (aka Navidrome) to test if the issue persists there as well? You could also ask on the Jellyfin community discord server to see if anyone else has experienced this.

**Reply by AECX** [reply]
Hey, sorry - Jellyfin is the only media system I use... An option to disable the default parameters would probably help, I can try to fiddle with it during vacation (mid summer) and, if successful, PR my changes.
